src/application.rs

```diff
@@ -34,6 +34,7 @@ pub enum ArgTypes {
 	BrightnessLower = 10,
 	NumLock = 11,
 	ScrollLock = 12,
+	TopMargin = 13,
```

Comment:
Move this down below `DeviceName` and set it's value to `isize::MIN + 1` for it to be executed before any other arg